### ISSUE TITLE
delete key sending 'Delete' text

### ISSUE THREAD

**Issue by AshlessCinder** [railsware/upterm#1145]
When using vim within upterm a delete keypress in insert mode writes the text `Delete` instead of removing a character like in other shells.

I'm running a new install of Antergos (just an installer for Arch) and Gnome.

**Comment by vlad-shatskyi** [comment]
Thanks for reporting, @AshlessCinder! I was able to reproduce it.

**Comment by vlad-shatskyi** [comment]
It's fixed in master and will be released soon. You can build it yourself, if you don't want to wait, it's pretty easy: just execute `npm run pack` in the project directory.

Also, do you mind if I ask you a few questions about your experience with Upterm?

**Comment by AshlessCinder** [comment]
Nice turn around!
And sure, ask away. I just started using it yesterday.

**Comment by vlad-shatskyi** [comment]
1. What features do you find the most valuable?
2. Is that something that irritates you?
3. What features do you miss?
4. How did you find out about Upterm?

**Comment by AshlessCinder** [comment]
1. definitely autocomplete of commands. I love how it defines the options for you (like -s or whatever) I think it could go further to allow newbies and everyone really to refresh themselves on what a command does and how it works without having to look up and grok a confusing help or manual page. [Like this](https://explainshell.com/) but not as in depth and more condensed
2. The slightly slow startup time, but it's not bad at all.
3. Not much TBH, but I'm not too much of a console jockey. I think having a persistent and transparent console that you hide and show with a hotkey like Guake or Tilda would be cool though. (this would also bypass the slower startup time)

I'd also say that having some customization options available via gui and/or configuration file would be nice.
4. I was searching around for electron based projects

Another idea that I would absolutely :heart: to see is a visual based `ls` command. It would make my console experience worlds better if I could easily see the filetypes as an icon instead of having to parse an extension.
Basically if the output of `ls` was shown similarly to a file explorer instead of a text list of filenames.

**Comment by vlad-shatskyi** [comment]
Thanks for your feedback, @AshlessCinder. We'll definitely have customization in the near future and will likely have an `ls` prettifier, but I doubt about a guake-like window.
